docs/adv-topics/usbinterfaces.md

```diff
@@ -9,6 +9,36 @@ Setting up USB audio interfaces is much easier with ASL3.
 
 The ASL3 menu and Asterisk CLI USB config commands handle these changes.
 
+### USB Interface IC Support
+
+Nearly all USB Interface products that are used with ASL nodes use C-Media CM108 and CM119 USB Interface ICs. These ICs provide the following features:
+
+ - Two 16-bit DAC audio outputs, capable of driving line level or headphone outputs.
+ - One 16-bit ADC mic/audio input with switchable mic preamp.
+ - Internal hardware mixers allowing ADC input and DAC output levels to be adjusted within a range of approximately 40dB, making it easy to optimally match input and output audio levels within the ASL tune utility settings, as well as a mixer allowing input audio to be mixed into output audio with zero latency.
+ - GPIO lines that support a PTT output, a Status LED and Clip LED, COS and CTCSS inputs, and one or more additional general-purpose I/O lines that can be controlled by ASL in response to events or DTMF commands.
+ - Support for an attached EEPROM IC to store configuration information such as a serial number.
```

Comment:
Can we simply say, "to store user configuration and for some chips manufacturer information".  For some devices the serial number information would be stored by the manufacturer.  I have been told by one chip distributor that the cm119B requires the EEPROM device to set some required defaults.

Reply:
I see no mention in the CM119B datasheet that an EEPROM is required. But I updated this line to also mention manufacturer information.